data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2020-02-03",
         },
     },
+    {
+        meta: {
+            name: "12/06/2020 Olympia",
+            link: "https://www.startnext.com/12062020",
+            keywords: ["petition", "networking", "event", "grassroots"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "One-time event where up to 90,000+ people come together to rent the Olympia stadium Berlin to sign various petitions."
+            + " In Germany, petitions must be discussed by parliament if they get more than 50,000 signatures. In addition there will be talks and presentations on currently pressing social and environmental problems.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
+            keywords: ["policy", "legislation"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "GermanZero is creating a climate action plan to meet Germany's 1.5C commitment that will be turned into proposed legislation before 2022. A nation-wide campaign will then try to get that legislation passed in parliament.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Hamburg muss handeln",
+            link: "http://hamburg-muss-handeln.de/",
+            keywords: ["private-sector", "legislation", "local", "grassroots"],
+            label: "hamburg-muss-handeln",
+        },
+        description: {
+            content: "A coalition of 300+ companies and organizations from Hamburg to express support for the demands of Fridays for Future. Also provides advice for upcoming state elections.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Fossil Free",
```

Comment:
This is an inititaive of 350.org https://350.org/
Please merge

Reply:
Let's use this to discuss and document what overlap we allow between initiatives, will then apply

Reply:
Due the fact that the fossil fuels groups are on the landing page and are one of the main aspects of 350.org I recommend to merge.